pages/work-ad.js

```diff
@@ -31,41 +33,13 @@ export default class WorkAd extends React.Component {
     const { ad } = this.props;
     return (
       <Layout>
-        <div className="hero-wrapper">
-          <div className="hero">
-            <div className="hero-content">
-              <h1>{ad.title}</h1>
-              <p>{ad.subtitle}</p>
-            </div>
-            <picture className="hero-media">
-              <source
-                media="(max-width: 400px)"
-                srcSet={mkUrl(ad.imageUrl)
-                  .width(800)
-                  .url()}
-              />
-              <source
-                media="(max-width: 799px)"
-                srcSet={mkUrl(ad.imageUrl)
-                  .width(800)
-                  .url()}
-              />
-              <source
-                media="(min-width: 800px)"
-                srcSet={mkUrl(ad.imageUrl).url()}
-              />
-              <img
-                className="hero-image"
-                role="presentation"
-                alt=""
-                src={mkUrl(ad.imageUrl)}
-              />
-            </picture>
-          </div>
-        </div>
+        <Hero {...ad} mkUrl={mkUrl} />
         <div className="jobAd">
-          <BlockContent blocks={ad.body} />
-          <a href={ad.externalLink}>apply here</a>
+          <div style={{ paddingRight: "80px" }}>
```

Comment:
Maybe it would be best to try and keep all styles for a component in the same place? To avoid confusion